### PR TITLE
Hotfix/735 Swapout check

### PR DESCRIPTION
**Pull request by Hoodathunk**
Mainly addresses Issue #735, regarding guardian modules being flagged for sell/re-buy.
1. Modules now matched using EDName, instead of EDDBID in script.
2. Fix module 'modified' flag in Journal Monitor & Frontier API code, updating to new FDev data format.
3. Added more guardian module definitions (reinforcements), misc undocumented sell prices, and general tidy up.

### CODE REVIEW

**Comment by Hoodathunk** [comment]
Prices found on ED Wiki.

File: ShipMonitor/FrontierApi.cs [hunk 1]
```diff
@@ -247,11 +247,8 @@ public static Module ModuleFromJson(string name, JObject json)
                 module.health = Math.Round(Health);
             }
 
-            // Flag if module has modifications
-            if (json["module"]["modifiers"] != null)
-            {
-                module.modified = true;
-            }
+            // Flag if module has engineering modifications
+            module.modified = json["engineer"] != null ? true : false;
```
Comment (richardbuckle) [inline ShipMonitor/FrontierApi.cs]: Unless I'm missing something, `!=` already produces a `bool` so we can elide `? true : false` here. But I'm equally happy to see it written out.

**Comment by richardbuckle** [comment]
Looks good but schedulining for 3.0,2 for risk management reasons. I think the following resx keys need to be added -- we can populate most languages from [Howard's spreadsheet](https://docs.google.com/spreadsheets/d/1GgWCZYH2w-XputyubcfVz-zH956gQLeMIs1E1XhYz1c/edit#gid=803697951)

- [ ] GuardianFsdBooster
- [ ] GuardianPowerDistributor
- [ ] EnzymeMissileRack
- [ ] ShardCannon

**Comment by richardbuckle** [comment]
Re-scheduling to 3.0.1 after discussion. Will add missing resx and translations.

File: ShipMonitor/FrontierApi.cs (hunk 1 above)
Comment (Hoodathunk) [reply ShipMonitor/FrontierApi.cs]: lol, you're ,of course, correct... I suppose I thought it read better.

**Comment by richardbuckle** [comment]
On detailed review, these were already present.

**Pull request merged**